### ISSUE TITLE
Testing models with predict.py does not give me any results file

### ISSUE THREAD

**Issue by mellahysf** [naver/sqlova#61]
I cant to test your project using predict.py,  i runned "python predict.py" commande with requires parameters and its still running in the step below without gives me the file of results. (in my cas results_dev.jsonl) ??
I'm tested on both windows and ubuntu (with CPU only and without CUDA).

XXXX@DESKTOP-YYYY:/mnt/c/users/administrateur/desktop/sqlova-master$ python3 predict.py --bert_type_abb uL --model_file models/model_best.pt --bert_model_file models/model_bert_best.pt --bert_path data --result_path result --data_path data --split dev
BERT-type: uncased_L-24_H-1024_A-16
Batch_size = 32
BERT parameters:
learning rate: 1e-05
Fine-tune BERT: False
vocab size: 30522
hidden_size: 1024
num_hidden_layer: 24
num_attention_heads: 16
hidden_act: gelu
intermediate_size: 4096
hidden_dropout_prob: 0.1
attention_probs_dropout_prob: 0.1
max_position_embeddings: 512
type_vocab_size: 2
initializer_range: 0.02
Seq-to-SQL: the number of final BERT layers to be used: 2
Seq-to-SQL: the size of hidden dimension = 100
Seq-to-SQL: LSTM encoding layer size = 2
Seq-to-SQL: dropout rate = 0.3
Seq-to-SQL: learning rate = 0.001

**Comment by Akshayextreme** [comment]
Try to run it for single question and single table first. Refer comments from `add_question.py` script.
